Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "B: With an exponential function with values from 0-5, what is the sigma value associated with x = 1? And associated with it, was is its probablity density?"
+    "B: With a standard exponential distrubution function with lambda = 1, what is the sigma value associated with x = 1?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C: In order to find sigma of x = 1, one must find the what's the value to yield a 50% cdf. Then subtract 1 from the 50% cdf divided by the standard deviation of a exponential distribution. Which is the square root of the (integral from 0 to infinity (x^2)e^-x ) - (integral from 0 to infinity xe^-x)^2. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 121,
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.6931471805599453\n"
+     ]
+    }
+   ],
+   "source": [
+    "# D:\n",
+    "print (stats.expon.ppf(.5))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "E: A special property of an exponential distrubution is that the mean is equal to the standard deviation. The mean is also 1/lambda, which is equal to 1. In order to calculate the sigma value of 1, subtract the x from the median divided by the SD. Which is 1-ln(2) / 1 = .31 SD. "
```

Comment:
Problem 3E:
- you can also do this in a line of code in addition to an explanation.
- typo: distrubution --> distribution